$NetBSD$

Has problems on systems that lack uselocale but have locale_t.

--- Makefile.libretro.orig	2017-03-29 18:42:18.000000000 +0000
+++ Makefile.libretro
@@ -85,7 +85,7 @@ ifneq (,$(findstring unix,$(platform)))
    TARGET := $(TARGET_NAME)_libretro.so
    fpic := -fPIC
    SHARED := -shared -Wl,-version-script=link.T
-   DEFINES += -std=c99 -D_GNU_SOURCE -DHAVE_LOCALE
+   DEFINES += -std=c99 -D_POSIX_C_SOURCE=200112L -D_XOPEN_SOURCE=600 -DHAVE_SETLOCALE
 
    # Raspberry Pi 3
    ifneq (,$(findstring rpi3,$(platform)))
